app/controllers/stories_controller.rb

```diff
@@ -351,22 +353,16 @@ def set_user_json_ld
         "@id": URL.user(@user)
       },
       "url": URL.user(@user),
-      "sameAs": [],
+      "sameAs": user_same_as,
       "image": ProfileImage.new(@user).get(width: 320),
       "name": @user.name,
-      "email": "",
-      "jobTitle": "",
+      "email": @user.email.presence,
+      "jobTitle": @user.employment_title.presence,
```

Comment:
How do we feel about this and the potential for `nil` values to be present here if the `email` and/or `employment_title` is not present?

Reply:
IIRC, JSON-LD doesn't allow for `nil` values? I played around with `nil` values in the structure [here](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%22http%3A%2F%2Fschema.org%22%2C%22email%22%3A%22vaidehi%40dev.to.%22%2C%22jobTitle%22%3A%22senior%20software%20engineer%22%7D), and I don't think `nil` values would work.

Reply:
That's what I was worried about @vaidehijoshi . 😕

Reply:
If a value doesn't exist, do you know if JSON-LD prefers an empty string or no key at all? If it prefers an empty string, I guess we can fallback to that (which is what it looks like you already had!), but if it doesn't even make use of empty strings, then we can just not include the key at all if its value is `nil`.

Reply:
No key it is!